mibidata/tiff.py

```diff
@@ -241,24 +242,36 @@ def read(file, sims=True, sed=False, optical=False, label=False):
             False.
         label: Boolean for whether to return the slide label image. Defaults to
             False.
+        inc_channels: List or tuple of channel names to include in MibiImage.
+                      Names should be string tuples of format (mass, target).
 
     Returns: A tuple of the image types set to True in the parameters, in the
         order SIMS, SED, Optical, Label (but including only those types
         specified). The SIMS data will be returned as a
         :class:`mibidata.mibi_image.MibiImage` instance; the other image
         types will be returned as numpy arrays. If an image type is selected to
         be returned  but is not present in the image, it will be returned as
-        None.
+        None. If returning SIMS data and the inc_channels parameter is set, only
+        those channels will be included in the MibiImage instance.
 
     Raises:
-        ValueError: Raised if the input file is not of the IonpathMIBI
-            format, or if no image type selected to be returned.
+        ValueError: Raised if
+
+            * The input file is not of the IonpathMIBI format, or if no image
+              type selected to be returned.
+            * The inc_channels parameter is not a tuple or list of
+              string tuples.
     """
     return_types = collections.OrderedDict([
         ('sims', sims), ('sed', sed), ('optical', optical), ('label', label)
     ])
     if not any((val for val in return_types.values())):
         raise ValueError('At least one image type must be specified to return.')
+    if (inc_channels and not
```

Comment:
If I'm reading this and the docstring correctly, `inc_channels` needs to be either a list of (mass, target) tuples or a single (mass, target) tuple, correct? 1) If that's the case, you should add tests for the case where it's a single tuple because right now the only ones are for 2 channels, which tests the list part. 2) Could this be generalized to pass in a single channel string or list of channel strings? I can imagine use cases for both.